netty/src/test/java/io/grpc/netty/NettyHandlerTestBase.java

```diff
@@ -222,4 +226,100 @@ protected final ByteBuf captureWrite(ChannelHandlerContext ctx) {
   protected abstract T newHandler() throws Http2Exception;
 
   protected abstract WriteQueue initWriteQueue();
+
+  protected abstract void makeStream() throws Exception;
```

Comment:
I made a separate method because CreateStream has different return types in ClientHandlerTest and ServerHandlerTest. Not sure if there is a way around this?